minio/console
# Use swagger api for import KSM Key

This also updates to use latest prettier version `3.2.4`

### Test Steps
1. Enable encryption like:
```
export MINIO_KMS_KES_ENDPOINT=https://play.min.io:7373                                                     
export MINIO_KMS_KES_KEY_FILE=root.key
export MINIO_KMS_KES_CERT_FILE=root.cert
export MINIO_KMS_KES_KEY_NAME=my-minio-sse-kms-key
```
2. Run minio
3. In UI go to Administration/Encryption
4. Click on Import Key
5. Type a new key and add bytes e.g. `2b7e151628aed2a6abf7158809cf4f3c`
6. New key should be added and listed
